spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/secretmanager/GcpSecretManagerAutoConfiguration.java

```diff
@@ -39,6 +41,7 @@
 @EnableConfigurationProperties(GcpSecretManagerProperties.class)
 @ConditionalOnClass(SecretManagerTemplate.class)
 @ConditionalOnProperty(value = "spring.cloud.gcp.secretmanager.enabled", matchIfMissing = true)
+@AutoConfigureAfter(GcpContextAutoConfiguration.class)
```

Comment:
I believe this was missed. Adding this allows beans defined in [GcpContextAutoConfiguration.java](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/core/GcpContextAutoConfiguration.java) be accessible in application.
In this case specifically, together with registerAndPromoteBean() in resolver, let application access the correct GcpProjectIdProvider by injecting it.